k4EDM4hep2LcioConv/include/k4EDM4hep2LcioConv/k4Lcio2EDM4hepConv.h

```diff
@@ -145,13 +145,6 @@ template <typename ObjectMappingT>
 std::vector<CollNamePair> createLinks(const ObjectMappingT& typeMapping,
                                       const std::vector<std::pair<std::string, EVENT::LCCollection*>>& LCRelation);
 
-template <typename ObjectMappingT>
-[[deprecated("use createLinks instead")]] std::vector<CollNamePair>
-createAssociations(const ObjectMappingT& typeMapping,
```

Comment:
Used in a docfile: https://github.com/key4hep/k4EDM4hep2LcioConv/blob/main/doc/LCIO2EDM4hep.md?plain=1#L131